app/scripts/TiledPixiTrack.js

```diff
@@ -756,18 +773,8 @@ class TiledPixiTrack extends PixiTrack {
         uuid: this.uuid,
       });
     }
-    const errors = Object.values(this.fetchedTiles)
-      .map(
-        (x) =>
-          x.tileData && x.tileData.error && `${x.tileId}: ${x.tileData.error}`,
-      )
-      .filter((x) => x);
 
-    if (errors.length) {
-      this.errorTextText = errors.join('\n');
-    } else {
-      this.errorTextText = '';
-    }
+    this.checkForErrors();
```

Comment:
Is this meant to be an inherited method? If not... I'd maybe recommend underscoring it (e.g., `_checkForErrors()`) to indicate an "internal" method.